Automattic/jetpack
# Update social-logos to include new reddit icon

I've used official Reddit asset and modified it a bit: removed background circle and cleared out all styling within svg file. These changes might need to be confirmed wih them. As per their brand book:

> Reddit permits a few specific uses without prior approval. For those uses, if you agree to the Reddit Brand Terms of Use and follow the guidelines in this document, you’re good to go. In most other cases, you’ll need to get our permission first.

Related PR for social-logos: https://github.com/Automattic/social-logos/pull/87

Fixes #9819

Internal reference: p7fD6U-2xE-p2

#### Changes proposed in this Pull Request:

* Updates social-logos assets to update style of reddit icon

#### Testing instructions:






* Make sure that sharing buttons are enabled for your site and reddit button is included. Select "Icon only" buttons type.
* Check any post, make sure sharing buttons are there, and reddit icon matches the acutal design.
* Change buttons type to "Icon & Text". Check the sharing button on your site.


One thing I not sure is the size of the icon. It looks fine in "Icon Only", but looks too big in "Icon & Text". Also `.sd-social-icon .sd-content ul li[class*='share-'] a.sd-button:before` class adds a `top: 1px` which put "Icons Only" icons off-center

**Before**

<img width="865" alt="Hello world! – HelloWord 2019-05-23 15-44-29" src="https://user-images.githubusercontent.com/5654161/58253709-af757e00-7d71-11e9-97a5-26f931842172.png">
<img width="1370" alt="before2" src="https://user-images.githubusercontent.com/5654161/58253711-b1d7d800-7d71-11e9-9d82-0c056c267b78.png">


**After**

<img width="1059" alt="Hello world! – HelloWord 2019-05-23 15-31-10" src="https://user-images.githubusercontent.com/5654161/58253370-f7e06c00-7d70-11e9-95cf-7c962ab0707c.png">
<img width="1356" alt="Hello world! – HelloWord 2019-05-23 15-35-00" src="https://user-images.githubusercontent.com/5654161/58253377-fa42c600-7d70-11e9-9fa2-0ef63b44088d.png">

#### Proposed changelog entry for your changes:

* update reddit icon
